docs/endpoints/multi_user.rst

```diff
@@ -1042,6 +981,235 @@ make the necessary changes to ``config.yaml``:
    $ globus-compute-endpoint configure my-mep --multi-user --auth-policy 2340174a-1a0e-46d8-a958-7c3ddf2c834a
 
 
+.. _high-assurance-mep:
+
+High-Assurance
+--------------
+
+Globus Compute endpoints may be designated as High-Assurance (HA) to meet stricter
+security, compliance, and operational requirements.  HA endpoints differ from non-HA
+endpoints in a few key ways:
+
+- in addition to running regular functions, HA endpoints may also run HA functions
+  (described below).
+
+- HA endpoints enable audit-logging, whereby the states of all tasks (whether HA or
+  not) are logged to a file.
+
+Consider deploying a High-Assurance endpoint where there is need for stronger identity
+verification and authentication controls, and for workloads that require elevated
+assurance levels, such as sensitive research or regulated data processing.
+
+.. note::
+
+   Once an endpoint has registered, the High-Assurance setting may not be toggled.
+   An endpoint may not become HA at a later date if it is not initially registered as
+   such.  The reverse ("downgrading" from HA) is similarly disallowed.
+
+
+Configuration
+^^^^^^^^^^^^^
+
+High-Assurance functionality requires a HA-enabled `Globus subscription`_, to be
+explicitly marked as HA, and to be associated with a HA policy.  In configuration form,
+that translates to ``subscription_id``, ``high_assurance``, and
+``authentication_policy``:
+
+.. code-block:: yaml
+   :caption: Example ``config.yaml`` of a HA endpoint
+
+   ...
+   subscription_id: 2c94f030-d346-11e9-939f-02ff96a5aa76
+   high_assurance: true
+   authentication_policy: 8e6529c8-a7ce-4310-b895-244e1b33702a
+   ...
+
+In this example, the ``subscription_id`` is associated with a HA-enabled subscription,
+and the ``authentication_policy`` has similarly been setup as HA.  Both of these values
+may be found in the Globus App `Web UI`_.  Find Subscriptions available to you via
+**Settings** |rarr| **Subscriptions**.
+
+Policies are associated with projects, so first navigate to **Settings**
+|rarr| **Developers** and select or create a project.  Within a project, navigate to
+the **Policies** tab.  If creating a policy, be sure to check the "High Assurance"
+checkbox.
+
+Alternatively, a new HA policy may be created at configuration time with an
+overly-specified command line:
+
+.. code-block:: console
+   :linenos:
+
+   # globus-compute-endpoint configure \
+       --multi-user \
+       --display-name "HA Documentation Compute Endpoint" \
+       --subscription-id 2c94f030-d346-11e9-939f-02ff96a5aa76 \
+       --high-assurance \
+       --auth-policy-project-id ba1e3f22-02d4-477b-b5d0-f8af2e8c419c \
+       --allowed-domains "globus.org" \
```

Comment:
I very much suggest not using 'globus.org' as an example placeholder for domain... because for most readers it would sound like a logical thing to include, so there is a huge chance people will use it verbatim and fail. `example.edu` per the [RFC](https://www.rfc-editor.org/rfc/rfc2606.html)

Similarly, I'd use fake presumptious `--subscription_id` and `--auth-policy-project-id` UUIDs - to make it more obvious that they need to be replaced.

As a side thought for this example, "documentation" does not seem that compute-intensive an illustration use case, so maybe just "HA Compute Endpoint"?

Reply:
All good points; implemented and pushed.  Naming wise, how about "Example High-Assurance Compute Endpoint"?

Reply:
Looks like there's still a leftover `ha_documentation_compute_endpoint` in there

Reply:
:facepalm: